Wikipedia/Code/WMFRandomSectionController.h

```diff
@@ -5,6 +5,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString* const WMFRandomSectionIdentifier = @"WMFRandomSectionIdentifier";
```

Comment:
@montehurd looks like this one (in the random section controller) still needs to be converted to an extern